src/services/destination/nativeIntegration.ts

```diff
@@ -113,8 +121,12 @@ export class NativeIntegrationDestinationService implements DestinationService {
           tags.FEATURES.ROUTER,
         );
         try {
+          const metadataWithSvcCtx = {
+            ...requestMetadata,
+            serviceContext: MiscService.getLoggableData(metaTO.errorDetails),
```

Comment:
this is more of request context right, service context might be misleading?

Reply:
The thought process behind naming was to reflect the context at service level.

Reply:
may be check with others and choose the one with less confusing